### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by Damix48**
My italian translation.

### CODE REVIEW

File: ui/extractedTranslations/it/translation.json [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+        "file_size": "Peso",
+        "file_type": "Formato",
+        "name": "Nome"
+      },
+      "title": "Download"
+    },
+    "media": {
+      "exif": {
+        "exposure_program": {
+          "action_program": "Programma azione",
```
Comment (Damix48) [inline ui/extractedTranslations/it/translation.json:194]: I'm not sure about this translation

File: ui/extractedTranslations/it/translation.json [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+          "aperture_priority": "Priorità apertura",
+          "bulb": "Bulb",
+          "creative_program": "Programma creativo",
```
Comment (Damix48) [inline ui/extractedTranslations/it/translation.json:197]: I'm not sure about this translation

File: ui/extractedTranslations/it/translation.json [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+          "landscape_mode": "Modalità orizzontale",
+          "manual": "Manuale",
+          "normal_program": "Programma normale",
+          "not_defined": "Non definito",
+          "portrait_mode": "Modalità verticale",
+          "shutter_priority": "Priorità otturatore"
+        },
+        "flash": {
+          "auto": "Auto",
+          "did_not_fire": "Non attivato",
```
Comment (Damix48) [inline ui/extractedTranslations/it/translation.json:207]: I'm not sure about this translation

File: ui/extractedTranslations/it/translation.json [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+          "fired": "Attivato",
```
Comment (Damix48) [inline ui/extractedTranslations/it/translation.json:208]: I'm not sure about this translation

File: ui/extractedTranslations/it/translation.json [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+          "no_flash": "No Flash",
+          "no_flash_function": "Nessuna funzione di flash",
+          "off": "Off",
+          "on": "On",
+          "red_eye_reduction": "Riduzione occhi rossi",
+          "return_detected": "Return detected",
```
Comment (Damix48) [inline ui/extractedTranslations/it/translation.json:214]: I don't know what this is

File: ui/extractedTranslations/it/translation.json [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+          "return_not_detected": "Return not detected"
```
Comment (Damix48) [inline ui/extractedTranslations/it/translation.json:215]: I don't know what this is

File: ui/extractedTranslations/it/translation.json (hunk 1 above)
Comment (viktorstrate) [reply ui/extractedTranslations/it/translation.json:194]: Action program is a setting many cameras have that is used to capture action, often sports and fast moving subjects.

File: ui/extractedTranslations/it/translation.json (hunk 2 above)
Comment (viktorstrate) [reply ui/extractedTranslations/it/translation.json:197]: Creative program is a camera setting that has creative features like black and white

File: ui/extractedTranslations/it/translation.json (hunk 3 above)
Comment (viktorstrate) [reply ui/extractedTranslations/it/translation.json:207]: If the flash did not fire

File: ui/extractedTranslations/it/translation.json (hunk 4 above)
Comment (viktorstrate) [reply ui/extractedTranslations/it/translation.json:208]: if the camera flash did fire (lighted up when the image was taken)

File: ui/extractedTranslations/it/translation.json (hunk 5 above)
Comment (viktorstrate) [reply ui/extractedTranslations/it/translation.json:214]: Whether the camera detected the light from the flash when taking the image

File: ui/extractedTranslations/it/translation.json (hunk 6 above)
Comment (viktorstrate) [reply ui/extractedTranslations/it/translation.json:215]: Opposite of the above

**Comment by viktorstrate** [comment]
Hi @Damix48 thank you for the translation. If you don't know how to translate a field, just set it to null, then the default english value will be used and it's clear that the translation is missing.

Here's an example of a missing field in the france translation:
https://github.com/photoview/photoview/blob/8cf4e8d779a9377c5d8b9cd125c09dafdbb1607f/ui/extractedTranslations/fr/translation.json#L66

File: ui/extractedTranslations/it/translation.json (hunk 1 above)
Comment (Damix48) [reply ui/extractedTranslations/it/translation.json:194]: Ok, in italian is often translate with `Programma sport` -> ,`Sport program`, do you prefer a more literally translation or it's fine to use `Sport`?

File: ui/extractedTranslations/it/translation.json (hunk 2 above)
Comment (Damix48) [reply ui/extractedTranslations/it/translation.json:197]: Ok, it's correct then

File: ui/extractedTranslations/it/translation.json (hunk 5 above)
Comment (Damix48) [reply ui/extractedTranslations/it/translation.json:214]: Thanks

File: ui/extractedTranslations/it/translation.json (hunk 4 above)
Comment (Damix48) [reply ui/extractedTranslations/it/translation.json:208]: Thanks

File: ui/extractedTranslations/it/translation.json (hunk 6 above)
Comment (Damix48) [reply ui/extractedTranslations/it/translation.json:215]: Thanks

File: ui/extractedTranslations/it/translation.json (hunk 3 above)
Comment (Damix48) [reply ui/extractedTranslations/it/translation.json:207]: Thanks

File: ui/extractedTranslations/it/translation.json (hunk 1 above)
Comment (viktorstrate) [reply ui/extractedTranslations/it/translation.json:194]: I think sport is fine

Comment (Damix48) [reply ui/extractedTranslations/it/translation.json]: Thank you

**Comment by Damix48** [comment]
> Hi @Damix48 thank you for the translation. If you don't know how to translate a field, just set it to null, then the default english value will be used and it's clear that the translation is missing.
> 
> Here's an example of a missing field in the france translation:
> https://github.com/photoview/photoview/blob/8cf4e8d779a9377c5d8b9cd125c09dafdbb1607f/ui/extractedTranslations/fr/translation.json#L66

Thanks

**Comment by Damix48** [comment]
I think the italian :it: translation is now complete.

**Comment by viktorstrate** [comment]
Awesome thank you for helping out :)

**Pull request merged**